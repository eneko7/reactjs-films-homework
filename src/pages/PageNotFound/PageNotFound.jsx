import React from 'react';
import style from './PageNotFound.scss';

const PageNotFound = () => (
  <div className={style.main_container_page_not_found}>
    <span className={style.main_container_page_not_found_span}>404 Error</span>
    <span className={style.main_container_page_not_found_span}>Page not found</span>
  </div>
);

export default PageNotFound;
