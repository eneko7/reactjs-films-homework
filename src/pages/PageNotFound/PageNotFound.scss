@font-face {
  font-family: "Helvetica_Neue";
  src: url('../../fonts/HelveticaNeueMedium.woff');
}

@font-face {
  font-family: "Helvetica";
  src: url('../../fonts/HelveticaNeue-Roman.woff');
}

.main_container_page_not_found {
  width: 70vw;
  height: 70vh;
  margin: 0 auto;
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
}

.main_container_page_not_found_span {
  font-size: 50px;
  color: #0aaee4;
  font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
  font-weight: 800;
}
