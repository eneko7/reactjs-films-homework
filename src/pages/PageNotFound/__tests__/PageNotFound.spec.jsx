import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import PageNotFound from '../index';

const shallow = new ShallowRenderer();

describe('PageNotFound Snapshot', () => {
  test('renders', () => {
    const component = shallow.render(
      <PageNotFound />,
    );
    expect(component).toMatchSnapshot();
  });
});
