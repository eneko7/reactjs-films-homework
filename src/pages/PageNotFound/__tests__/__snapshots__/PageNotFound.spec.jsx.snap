// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Snapshot renders 1`] = `
<div
  className="main_container_page_not_found"
>
  <span
    className="main_container_page_not_found_span"
  >
    404 Error
  </span>
  <span
    className="main_container_page_not_found_span"
  >
    Page not found
  </span>
</div>
`;
